diff --git a/node_modules/react-qr-reader/dist/index.js b/node_modules/react-qr-reader/dist/index.js
index 92c2f48..5873a43 100644
--- a/node_modules/react-qr-reader/dist/index.js
+++ b/node_modules/react-qr-reader/dist/index.js
@@ -140,7 +140,7 @@
         };
         return (React.createElement(React.Fragment, null,
-            React.createElement("video", { ref: function (ref) { return (_this.video = ref); }, style: __assign({ width: "100%", height: "100%" }, videoStyle), className: className }),
+            React.createElement("video", { ref: function (ref) { return (_this.video = ref); }, style: __assign({ width: "100%", height: "100%" }, videoStyle), className: className, muted: true, autoPlay: true, playsInline: true }),
             children));
     };
     QrReader.prototype.componentDidMount = function () {
diff --git a/node_modules/react-qr-reader/dist/index.d.ts b/node_modules/react-qr-reader/dist/index.d.ts
index c8d99ad..32a51f8 100644
--- a/node_modules/react-qr-reader/dist/index.d.ts
+++ b/node_modules/react-qr-reader/dist/index.d.ts
@@ -1,5 +1,6 @@
 /// <reference types="react" />
 import * as React from "react";
+import { Result } from '@zxing/library';
 declare type FacingMode = "user" | "environment";
 declare type TorchState = "on" | "off";
 export interface QrReaderProps {
@@ -9,7 +10,7 @@
     onScan: (result: string | null) => void;
     /**
      * @deprecated
-     * facingMode will be removed in the future, use constraints instead
+     * `facingMode` will be removed in the future, use constraints instead
      *
      * @see https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Syntax
      */
@@ -21,7 +22,7 @@
     style?: any;
     className?: string;
     onLoad?: () => void;
-    onError?: (error: any) => void;
+    onError: (error: any) => void;
     /**
      *
      * By default, the camera is only requested when the component is mounted.
@@ -34,7 +35,7 @@
      */
     showViewFinder?: boolean;
 }
-export declare class QrReader extends React.Component<QrReaderProps, {
+export declare class QrReader extends React.Component<QrReaderProps, {
     loading: boolean;
     mirrorVideo: boolean;
 }> {
@@ -47,7 +48,7 @@
     private video;
     private lastSelectedDeviceId;
     private mounted;
-    constructor(props: any);
+    constructor(props: QrReaderProps);
     componentDidMount(): void;
     componentWillUnmount(): void;
     private onVideoFrame;
@@ -58,6 +59,6 @@
     private handleVideo;
     private handleLoad;
     private openImageDialog;
-    render(): JSX.Element;
+    render(): React.ReactNode;
 }
 export {};
diff --git a/node_modules/react-qr-reader/package.json b/node_modules/react-qr-reader/package.json
index 548a514..5f9c566 100644
--- a/node_modules/react-qr-reader/package.json
+++ b/node_modules/react-qr-reader/package.json
@@ -27,11 +27,11 @@
     "build": "tsc"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react": "*",
+    "react-dom": "*"
   },
   "dependencies": {
     "@zxing/library": "^0.18.6",
     "webrtc-adapter": "^8.1.1"
   }
-}
+}
\ No newline at end of file
